native/machine-coding/file-uploader/style.css

```diff
@@ -0,0 +1,114 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  
+}
+
+
+::selection{
+  color: #fff;
+  background: #6990F2;
+}
+.wrapper{
+  /* width: 430px; */
```

Comment:
Since you have removed width completely, I think now this wrapper will occupy full screen width in desktop. This wont look good.

Instead of removing the width entirely, you can use max-width. In that way the wrapper will have 430px width in larger screens, and it won't overflow also in smaller screens.

Also you can add `margin: auto` to bring the wrapper in the center after giving `max-width: 430px`.